### ISSUE TITLE
pysolfc 2.1.2 is broken

### ISSUE THREAD

**Issue by ncmprhnsbl** [voidlinux/void-packages#10713]
### System

* xuname:  
Void 4.14.12_4 x86_64 GenuineIntel uptodate rrFFF
* package:  
pysolfc 2.1.2

### Expected behavior
work
### Actual behavior
doesn't work
### Steps to reproduce the behavior
```
 $  pysol
Traceback (most recent call last):
  File "/usr/bin/pysol", line 27, in <module>
    from pysollib.main import main
  File "/usr/lib/python2.7/site-packages/pysollib/main.py", line 33, in <module>
    from pysollib.util import DataLoader
  File "/usr/lib/python2.7/site-packages/pysollib/util.py", line 30, in <module>
    from pysollib.mfxutil import Image
  File "/usr/lib/python2.7/site-packages/pysollib/mfxutil.py", line 30, in <module>
    from six import print_
ImportError: No module named six
```

**Comment by Hoshpak** [comment]
Please try if installing python-six makes it work. If it does, we can just add this package as runtime dependency. I can't test it myself right now since that package is already installed on all of my systems.

**Comment by ebfe** [comment]
https://github.com/voidlinux/void-packages/pull/10595#issuecomment-357221541

**Comment by wpbirney** [comment]
this is fixed with #10726 being merged

**Comment by ncmprhnsbl** [comment]
pysolfc now updated with new deps:
```
python-six
python-random2
pysolfc-cardsets
```
working fine here, thanks guys :)
